save-backend/backend-api-docs.json

```diff
@@ -1693,10 +1693,10 @@
         ]
```

Comment:
May be make this job scheduled instead of commiting in any PR?
For example, every day at 5AM or something like this?

cc @nulls

Reply:
let's create an issue for it